run_dnlm_flax.py

```diff
@@ -373,7 +372,7 @@ def tokenize_function(examples):
     # e.g: {"input_ids": [[1,2,3,4]]} -> {"input_ids": [1,2,3,4]}
     def flatten(example):
         for k, v in example.items():
-            example[k] = v[0]
+            example[k] = v
```

Comment:
I'm not sure about this, when I'm running with pile, this needs to flatten, did you encounter the error when run with wikitextdata?
```suggestion
			if isinstance(v[0], list):
                example[k] = v[0]
            else:
                example[k] = v

```
This might be a better solution for both cases I guess 🤔 . What do you think @tylerjthomas9 ?